config.json

```
@@ -3,9 +3,9 @@
   "timeframe": "5m",
   "max-open-trades": 3,
   "starting-capital": 1000,
-  "backtesting-from": "2021-03-11",
-  "backtesting-to": "2021-03-15",
-  "backtesting-till-now": false,
+  "backtesting-from": "2021-06-05",
+  "backtesting-to": "2021-06-15",
+  "backtesting-till-now": true,
```

Don't push your changes to the config file template!
```suggestion
  "backtesting-from": "2021-03-11",
  "backtesting-to": "2021-03-15",
  "backtesting-till-now": false,
```